### DISCUSSION TITLE
Can I set different overrided dependency versions for each triplet?

### DISCUSSION THREAD

**Discussion by works-out** [microsoft/vcpkg#33556] [Q&A]
I want to use vcpkg to cross platform project.
This project uses different OpenSSL version by historical reason.
How can I set each version in vcpkg.json at overrides section?

For example, I want to use OpenSSL 3.x on Windows and OpenSSL 1.x on Linux.

**Comment by autoantwort** [comment]
Afaik this is not possible. You could use multiple manifest files as a workaround.
